.github/workflows/newman-action-run.yml

```diff
@@ -0,0 +1,17 @@
+name: newman action run
+
+on: pull_request
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Run newman
+        uses: matt-ball/newman-action@master
+        with:
+          apiKey: ${{ secrets.POSTMAN_API_KEY }}
+          collection: ${{ secrets.POSTMAN_COLLECTION_ID }}
+          environment: ${{ secrets.POSTMAN_ENVIRONMENT_ID }}
```

Comment:
newman-action has a bug in the case that environment is not set.
https://github.com/matt-ball/newman-action/pull/45